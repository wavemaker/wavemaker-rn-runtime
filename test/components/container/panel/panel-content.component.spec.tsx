import React, { ReactNode } from 'react';
import { Text, TouchableOpacity } from 'react-native';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import WmPanelContent from '@wavemaker/app-rn-runtime/components/container/panel/panel-content/panel-content.component';

describe('Test PanelContent component', () => {
    test('Check validity of sample component', () => {
      const tree = renderer.create(<WmPanelContent name="test_PanelContent"/>).toJSON();
      expect(tree).toMatchSnapshot();
    });
});