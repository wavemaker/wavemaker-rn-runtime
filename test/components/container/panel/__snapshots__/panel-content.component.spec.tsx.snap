// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test PanelContent component Check validity of sample component 1`] = `
<View
  style={
    {
      "borderColor": "#ffffff",
      "borderStyle": "solid",
      "borderTopWidth": 0,
      "borderWidth": 1,
    }
  }
/>
`;
